src/api/Storage.ts

```diff
@@ -73,7 +82,18 @@ export class GlobalStorage extends Storage {
   async setLastConnections(lastConnections: LastConnection[]) {
     await this.set("lastConnections", lastConnections.sort((c1, c2) => c2.timestamp - c1.timestamp));
   }
-}
+
+  getServerSettingsCache(name: string) {
+    return this.get<CachedServerSettings>(SERVER_SETTINGS_CACHE_KEY.concat(`_${name}`));
```

Comment:
Continued from suggestion's above:
```suggestion
    return this.get<CachedServerSettings>(SERVER_SETTINGS_CACHE_KEY(name));
```